site/content/en/guides/producer/functions/ts/develop/_index.md

```diff
@@ -0,0 +1,253 @@
+
+```sh
+npm run kpt:docker-push
+```
+
+You'll need proper authentication/authorization to push to your registry.
+
+`kpt:docker-push` pushes to the registry specified in the `kpt.docker_repo_base` field in `package.json`.
+You can manually edit this field at any time.
+
+The default value for the container image tag is `dev`. This can be overridden using`--tag` flag:
+
+```sh
+npm run kpt:docker-build -- --tag=latest
+npm run kpt:docker-push -- --tag=latest
+```
+
+## Use the SDK CLI
+
+The `create-kpt-functions` package (installed as `devDependencies`), provides a CLI for managing
+the NPM package you created above. The CLI sub-commands can be invoked via `npm run`. For example,
+to add a new function to the package:
+
+```console
+npm run kpt:function-create
+```
+
+These sub-commands are available:
+
+```console
+kpt:docker-create       Generate Dockerfiles for all functions. Overwrite
+                        files if they exist.
+kpt:docker-build        Build container images for all functions.
+kpt:docker-push         Push container images to the registry for all functions.
+kpt:function-create     Generate stubs for a new function. Overwrites files
+                        if they exist.
+kpt:type-create         Generate classes for core and CRD types. Overwrite
+                        files if they exist.
+```
+
+Flags are passed to the CLI after the `--` separator. For example, to pass a tag
+when building a container image:
+
+```console
+npm run kpt:docker-build -- --tag=latest
+```
+
+## Next Steps
+
+- Learn how to [run functions].
+- Find out how to structure a pipeline of functions from the [functions concepts] page.
+- Take a look at [these example functions][demo-funcs] to better understand how to use the typescript SDK.
+
+[spec]: https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/api-conventions/functions-spec.md
+[kustomize-run]: https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/api-conventions/functions-impl.md
+[demo-funcs]: https://github.com/GoogleContainerTools/kpt-functions-sdk/tree/master/ts/demo-functions/src
+[label-namespace]: https://github.com/GoogleContainerTools/kpt-functions-sdk/tree/master/ts/demo-functions/src/label_namespace.ts
+[catalog]: ../../../../consumer/function/catalog
+[api-kptfunc]: https://googlecontainertools.github.io/kpt-functions-sdk/api/interfaces/_types_.kptfunc.html
+[vscode]: https://code.visualstudio.com/
+[npm-packages]: https://github.com/GoogleContainerTools/kpt-functions-sdk/packages
+[download-node]: https://nodejs.org/en/download/
+[download-kind]: https://github.com/kubernetes-sigs/kind
+[install-node]: https://github.com/nodejs/help/wiki/Installation
+[install-docker]: https://docs.docker.com/v17.09/engine/installation
+[beta-feature]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.15.md#customresourcedefinition-openapi-publishing
+[run functions]: ../../../../consumer/function/
```

Comment:
Can you double check there are no broken links (Changes are high with such moves). Maybe also look into whether there's an automated way to catch broken links.

Reply:
I installed and used muffet locally. https://github.com/raviqqe/muffet
I also ran the W3 link checker on the existing kpt, kpt-functions-sdk, and kpt-functions-catalog websites and fixed some of the links it caught.